app/javascript/application.js

```diff
@@ -7,6 +7,8 @@ import $ from 'jquery'
 
 initAll()
 
+// Most of this Turbo setup code is needed for the multi-form gem
```

Comment:
Sorry, I wasn't clear (I blame covid), and Turbo isn't very clear. So: Turbo comprises a couple of different things, including Turbolinks and Turboframes. The config here is basically trying to turn Turbolinks off because it's a plague and an abomination, but you're absolutely right, it does seem like removing the whole library is a bit more of a task than I naively thought (because, e.g. the cookie banner seems to rely on Turboframes still).

BUT the bit in application.js that I was referring to is the bit at the very bottom, line 55 onwards, which is specifically about the turboframes code that you've just removed from the views. So _that_ stuff can safely go, because otherwise it's orphaned JS that no longer does anything.

Reply:
I tried removing just that and the tests still failed though

Reply:
That seems quite suspect - the code only does anything if there's HTML with CSS selector `.reload-visible-turboframes`, and you just removed the only instance of that class.

Reply:
Either that or I removed more than I thought. It passed locally so let's see what CI says